src/test/java/com/microsoft/sqlserver/UnitStatement/CallableMixedTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Microsoft JDBC Driver for SQL Server
+ * 
+ * Copyright(c) Microsoft Corporation All rights reserved.
+ * 
+ * This program is made available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+package com.microsoft.sqlserver.UnitStatement;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.microsoft.sqlserver.testframework.AbstractSQLGenerator;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+import com.microsoft.sqlserver.testframework.util.RandomUtil;
+
+@RunWith(JUnitPlatform.class)
+public class CallableMixedTest extends AbstractTest {
+    Connection connection1 = null;
+    Statement Statement151 = null;
+    String tableN = RandomUtil.getIdentifier("TFOO3");
+    String procN = RandomUtil.getIdentifier("SPFOO3");
+    String tableName = AbstractSQLGenerator.escapeIdentifier(tableN);
+    String procName = AbstractSQLGenerator.escapeIdentifier(procN);
+
+    /**
+     * Tests Callable mix
+     * @throws SQLException
+     */
+    @Test
+    @DisplayName("Test CallableMix")
+    public void datatypestest() throws SQLException {
+        connection1 = DriverManager.getConnection(connectionString);
+        Statement151 = connection1.createStatement();
```

Comment:
All instance variable's name should be in **lowerCamelCase** while static final variable names should be in **CAPS**.